src/subprocess-win32.cc

```diff
@@ -107,7 +107,9 @@ bool Subprocess::Start(SubprocessSet* set, const string& command) {
 
   // Do not prepend 'cmd /c' on Windows, this breaks command
   // lines greater than 8,191 chars.
-  if (!CreateProcessA(NULL, (char*)command.c_str(), NULL, NULL,
+  std::wstring commands = Utf8ToWide(command).c_str();
```

Comment:
In addition to the use of C++11, and .data() being acceptable, you're also calling Utf8ToWide, which returns a std::wstring, but then calling c_str() on it, and then constructing a new std::wstring from that wchar_t*, which

1. Requires a new allocation
2. Requires wstring's constructor to find the nul terminator (e.g. std::strlen)
3. Requires copying every byte of the string.

Better to simply do

```cpp
std::wstring wcommand = Utf8ToWide(command);
```

Reply:
Better still, do it in a single line

```cpp
if (!CreateProcessW(NULL, Utf8ToWide(command).data(), NULL, NULL,
```